benchmarks/dynamo/common.py

```diff
@@ -1858,6 +1858,7 @@ def run(runner, args, original_dir=None):
         args.ci = True
     if args.dynamic_shapes:
         torch._dynamo.config.dynamic_shapes = True
+        torch._functorch.config.use_dynamic_shapes = True
```

Comment:
This is not supposed to be necessary post https://github.com/pytorch/pytorch/pull/94469 ; can we manually prop it to the recursive aot autograd call instead?